easybuild/easyconfigs/p/pgi/pgi-17.10-gcc-5.4.0-2.26.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'Bundle'
+
+name = 'PGI'
+version = '17.10'
+gccver = '5.4.0'
+binutilsver = '2.26'
+versionsuffix = '-GCC-%s-%s' % (gccver, binutilsver)
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+dependencies = [
+    ('gcc/5.4.0-2.26', EXTERNAL_MODULE),
```

Comment:
Would it make it more easy to maintain this, if the variables defined above were also used for the dependencies?

Reply:
this module is provided by EB, why are you using as external module?

Reply:
this is what i get if i don't put it as external:
```
[kesch] ERROR: Irresolvable dependencies encountered: gcc/5.4.0-2.26
```

`/apps/escha/UES/PrgEnv-gnu-17.02/modulefiles/gcc/5.4.0-2.26`

Reply:
GCC is capital in EB.

MCH asked to have a lowercase naming scheme so the module names are misleading, we need always to look at the eb files.